src/app/public/modules/description-list/description-list.component.scss

```diff
@@ -0,0 +1,142 @@
+@import "~@skyux/theme/scss/themes/modern/_compat/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+@import "~@skyux/theme/scss/mixins";
+
+.sky-description-list {
+  margin: 0;
+  padding: 0;
+
+  dd,
+  dt {
+    margin: 0;
+    padding: 0;
+  }
+
+  // Name-value pair mode
```

Comment:
```suggestion
  // Standard pair mode
```

Reply:
@Blackbaud-AlexKingman, should we make this change for consistency in our terminology?